packages/nextjs/app/builders/layout.tsx

```diff
@@ -0,0 +1,12 @@
+import { getMetadata } from "~~/utils/scaffold-eth/getMetadata";
+
+export const metadata = getMetadata({
+  title: "Buidlers batch #4",
```

Comment:
Maybe just keep it "Builders" ?  Sinc we are using templates in root layout.tsx the `batch # 4` feels a bit repetitive : 

![Screenshot 2024-04-14 at 9 37 19 PM](https://github.com/BuidlGuidl/batch4.buidlguidl.com/assets/80153681/a36ec47e-6daa-4158-b9c9-fe12b9451149)

Reply:
Done @technophile-04